ros2bag/ros2bag/verb/play.py

```diff
@@ -43,4 +46,5 @@ def main(self, *, args):  # noqa: D102
             uri=bag_file,
             storage_id=args.storage,
             node_prefix=NODE_NAME_PREFIX,
-            read_ahead_queue_size=args.read_ahead_queue_size)
+            read_ahead_queue_size=args.read_ahead_queue_size,
+            exclude_topics=args.exclude_topics)
```

Comment:
The check for unique topic names can be done in python. I guess that's a bit easier and avoids the need of a `std::set`. 
Something like `exclude_topics = list(set(args.exclude_topics))`

Reply:
For me a set would be the natural choice of container. I guess at this point it doesn't make a difference, but the idea is to have a O(log n) complexity for excluding topic names. 

Using a set on python side, wouldn't guarantee, that the API isn't misused from Python side in the future. Using a sorted vector and binary search for the topic lookup bloats the code. @Karsten1987, are you sure you want to have a std::vector in favor of a std::set?